falcon/asgi/request.py

```diff
@@ -699,7 +700,7 @@ def netloc(self):
 
         return netloc_value
 
-    async def get_media(self, default_when_empty=_UNSET):
+    async def get_media(self, default_when_empty=_UNSET, _shortcut_cache={}):
```

Comment:
Giving this pattern a try. A sphinx extension is used to filter it out from the docs.